sources/vega/vega.proto

```diff
@@ -1139,6 +1139,8 @@ message EpochData {
 }
 
 message Node {
+  reserved 5;
```

Comment:
Do not bother for now with the reserved field, we'll do that once the API is stable.